validate/validate.proto

```diff
@@ -1,6 +1,8 @@
 syntax = "proto2";
 package validate;
 
+option go_package = "github.com/lyft/protoc-gen-validate/validate";
```

Comment:
It looks like this may have confused the make target for generating `validate.pb.go`